config/environments/production.rb

```diff
@@ -52,7 +52,8 @@
 
   # Include generic and useful information about system operation, but avoid logging too much
   # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :error
+  config.log_level = ENV.fetch('LOG_LEVEL') { 'info' }.downcase.to_sym
```

Comment:
Our Rubocop config will complain about the syntax — it prefers that we write:

```ruby
ENV.fetch('LOG_LEVEL', 'info').downcase.to_sym
```

I ran into this as well on my PR. 😅

Reply:
Oh, I've not checked Rubocop locally

Reply:
You should see a GitHub Actions failure in a few minutes. The STDOUT of the syntax check test will show the error.